### DISCUSSION TITLE
Twitter | System Design

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#47] [Comments]
# Twitter | System Design

In this tutorial, we will learn how to design Twitter

https://www.karanpratapsingh.com/courses/system-design/twitter

**Comment by haobird** [comment]
Hello , which app you draw Data model design ?

**Reply by karanpratapsingh** [reply]
Hi @haobird, I used [drawSQL](https://drawsql.app/) for data model diagrams.
